extensions/azurecore/src/azureResource/tree/flatAccountTreeNode.ts

```diff
@@ -178,13 +178,25 @@ class FlatAccountTreeNodeLoader {
 			}
 		}, 500);
 		try {
+
+			// Authenticate to tenants to filter out subscriptions that are not accessible.
+
+			let tenants = this._account.properties.tenants;
+			if (tenants.length !== 0) {
+				// Filter out tenants that we can't authenticate to.
+				tenants = tenants.filter(async tenant => {
+					const token = await azdata.accounts.getAccountSecurityToken(this._account, tenant.id, azdata.AzureResource.ResourceManagement);
```

Comment:
Consider adding logging here for tenants we couldn't authenticate to (unless the logging in getAccountSecurityToken already gives that info)

Reply:
getAccountSecurityToken takes care of logging.